integration-tests/appsec/esm-app/worker-dep.mjs

```diff
@@ -1,5 +1,3 @@
-'use strict'
-
```

Comment:
wait why remove it ?

Reply:
This is an mjs file (i.e. an ES Module), which means that strict mode is enabled by default. No need to manually set it like this